train.py

```diff
@@ -198,18 +200,20 @@ def train(self):
 
     best_valid_loss = 1.e6
     for epoch in range(self.startEpoch, self.params.max_epochs):
+      logging.info(f"Epoch: {epoch+1}")
       if dist.is_initialized():
         self.train_sampler.set_epoch(epoch)
 #        self.valid_sampler.set_epoch(epoch)
 
       start = time.time()
       tr_time, data_time, train_logs = self.train_one_epoch()
       valid_time, valid_logs = self.validate_one_epoch()
+      inference_logs = self.inference_one_epoch()
+      del tr_time, data_time, valid_time
```

Comment:
is this necessary, or just that these variables are unused?

Reply:
These variables are unused. See comment above.